rest/model/data/meta.go

```diff
@@ -442,6 +458,33 @@ func validateNoteLength(v reflect.Value) error {
 	return nil
 }
 
+func validatePulsar(v reflect.Value) error {
+	var pulsars []*PulsarMeta
+
+	if v.Kind() == reflect.Slice {
+		// Slice from API
+		bs, err := json.Marshal(v.Interface())
+		if err != nil {
+			return fmt.Errorf("pulsar: unexpected value: `%v`", v.Interface())
+		}
+		if err := json.Unmarshal(bs, &pulsars); err != nil {
+			return fmt.Errorf("pulsar: invalid value: `%v`", v.Interface())
+		}
+	} else if v.Kind() == reflect.String {
+		// String from terraform
+		if err := json.Unmarshal([]byte(v.String()), &pulsars); err != nil {
+			return fmt.Errorf("pulsar: invalid value: `%v`", v.String())
+		}
+	}
```

Comment:
really wanted to not duplicate the Unmarshal calls but failed miserably at it :/